### ISSUE TITLE
Arch-specific builds contain constants for all architectures

### ISSUE THREAD

**Issue by copy** [AlexAltea/capstone.js#7]
On my minor conquest to reduce the size of `capstone-x86.min.js`, I noticed that the `cs` object exported by `capstone-x86.min.js` also contains constants for other architectures (`ARM_*`, `MIPS_*`, etc.).

I believe these could be safely removed to achieve a decent (about 12k) size reduction.

**Comment by AlexAltea** [comment]
True, constants were loaded from [capstone-constants.js](https://github.com/AlexAltea/capstone.js/blob/master/src/capstone-constants.js) regardless of what was actually being built.

I think the build scripts need to be updated to take care of this issue, and the previous one you reported (#6, which by the way would also reduce the amount of constants, as far as I know).

I would gladly do this immediately, but I will be quite busy for the next weeks, so any pull requests would be really welcome. :-)

**Comment by AlexAltea** [comment]
This is now fixed in Capstone.js v5.0.9: https://github.com/AlexAltea/capstone.js/releases/tag/v5.0.9

We have now one `constants_<arch>.js` file per architecture.
